common/src/main/java/com/wynntils/models/territories/GuildAttackTimerModel.java

```diff
@@ -50,8 +50,9 @@
 public final class GuildAttackTimerModel extends Model {
     private static final Pattern GUILD_ATTACK_PATTERN = Pattern.compile("§b- (.+):(.+) §3(.+)");
     private static final Pattern GUILD_DEFENSE_CHAT_PATTERN = Pattern.compile("§3.+§b (.+) defense is (.+)");
+    // Test in GuildAttackTimerModel_WAR_MESSAGE_PATTERN
     private static final Pattern WAR_MESSAGE_PATTERN = Pattern.compile(
-            "§3\\[WAR\\]§c The war for (?<territory>.+) will start in (?<remaining>.+) (?<type>minutes|seconds)\\.");
+            "§c(?:\uDAFF\uDFFC\uE006\uDAFF\uDFFF\uE002\uDAFF\uDFFE|\uDAFF\uDFFC\uE001\uDB00\uDC06) The war for (?<territory>.+)(?: ?\\n\uDAFF\uDFFC\uE001\uDB00\uDC06)? will(?: ?\\n\uDAFF\uDFFC\uE001\uDB00\uDC06)? start(?: ?\\n\uDAFF\uDFFC\uE001\uDB00\uDC06)? in(?: ?\\n\uDAFF\uDFFC\uE001\uDB00\uDC06)? ((?<minutes>\\d+)(?: ?\\n\uDAFF\uDFFC\uE001\uDB00\uDC06)? minute(?:s)?)?(?:(?: ?\\n\uDAFF\uDFFC\uE001\uDB00\uDC06)? and )?((?<seconds>\\d+)(?: ?\\n\uDAFF\uDFFC\uE001\uDB00\uDC06)? second(?:s)?)?\\.");
```

Comment:
What's up with all these alignment characters..? Maybe StyledText should have a method for removing them? Maybe like `trimAlignment`?

Reply:
Okay, since this is the only thing holding this PR back, I'll add this shortly.